vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -488,11 +488,6 @@ - (void)listRecurringTips:(void (^)(NSArray<BATPublisherInfo *> *))completion
   });
 }
 
-- (void)addRecurringTipToPublisherWithId:(NSString *)publisherId amount:(double)amount
-{
-  ledger->AddRecurringPayment(std::string(publisherId.UTF8String), amount);
```

Comment:
And instead of removing this entire method, we'll replace this call with:

```objc
  [BATLedgerDatabase insertOrUpdateRecurringTipWithPublisherID:publisherId
                                                        amount:amount
                                                     dateAdded:[[NSDate date] timeIntervalSinceNow]
                                                    completion:nil];
```

Same way that `rewards_service_impl` handles this in `SaveRecurringTip()`